src/@commonsku/styles/Popup.tsx

```diff
@@ -95,28 +95,29 @@ const PopupContainer: React.FC<{}> = ({ children }) => {
   return ReactDOM.createPortal(children, ref.current);
 }
 
-export type PopupProps = React.PropsWithChildren<{ 
-    header?: React.Component,
-    noHeader?: boolean,
-    title?: string|React.Component,
-    controls?: Array<React.ReactNode>,
-    onClose?: (event?: React.MouseEvent) => void,
-    noCloseButton?: boolean,
-    closeOnClickOutside?: boolean,
-    closeOnEsc?: boolean,
-    width?: string,
-    height?: string,
-    padding?: string,
-    zIndex?: number,
-    overlayZIndex?: number;
-    popupClassName?: string;
-    contentClassName?: string;
+export type PopupProps = React.PropsWithChildren<{
+  header?: React.Component,
+  noHeader?: boolean,
+  title?: string | React.Component,
+  controls?: Array<React.ReactNode>,
+  onClose?: (event?: React.MouseEvent) => void,
+  noCloseButton?: boolean,
+  closeOnClickOutside?: boolean,
+  closeOnEsc?: boolean,
+  width?: string,
+  height?: string,
+  padding?: string,
+  zIndex?: number,
+  overlayZIndex?: number;
+  popupClassName?: string;
+  contentClassName?: string;
+  backgroundComponent?: React.ReactNode,
```

Comment:
Maybe instead of passing additional backgroundComponent,  we could allow override PopupWindow